### PR TITLE
[GEARPUMP-343] Fix typo of EmbeddedRuntimeEnvironment

### PR DESCRIPTION
**Pull request by huafengw**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the commit message is formatted like:
   `[GEARPUMP-<Jira issue #>] Meaningful description of pull request` 
 - [ ] Make sure tests pass via `sbt clean test`.
 - [x] Make sure old documentation affected by the pull request has been updated and new documentation added for new functionality.

### CODE REVIEW

**Comment by huafengw** [comment]
```
The job exceeded the maximum time limit for jobs, and has been terminated.
```

File: core/src/main/scala/org/apache/gearpump/cluster/embedded/EmbeddedRuntimeEnvironment.scala [hunk 1]
```diff
@@ -19,20 +19,20 @@ package org.apache.gearpump.cluster.embedded
 
 import com.typesafe.config.Config
 import org.apache.gearpump.cluster.client.{ClientContext, RuntimeEnvironment}
-import org.apache.gearpump.cluster.embedded.EmbeddedRuntimeEnvironemnt.EmbeddedClientContext
+import org.apache.gearpump.cluster.embedded.EmbeddedRuntimeEnvironment.EmbeddedClientContext
 
 /**
  * The EmbeddedRuntimeEnvironemnt is initiated when user trying to launch their application
```
Comment (manuzhang) [inline core/src/main/scala/org/apache/gearpump/cluster/embedded/EmbeddedRuntimeEnvironment.scala]: Do not forget the comments.

File: core/src/main/scala/org/apache/gearpump/cluster/embedded/EmbeddedRuntimeEnvironment.scala [hunk 2]
```diff
@@ -19,20 +19,20 @@ package org.apache.gearpump.cluster.embedded
...
  * from IDE. It will create an embedded cluster and user's applcaition will run in a single
```
Comment (manuzhang) [inline core/src/main/scala/org/apache/gearpump/cluster/embedded/EmbeddedRuntimeEnvironment.scala]: also "applcaition"

**Comment by manuzhang** [comment]
LGTM

**Pull request closed without merging**